system/jhtree/jhtree.cpp

```diff
@@ -1113,13 +1117,14 @@ CDiskKeyIndex::CDiskKeyIndex(unsigned _iD, IFileIO *_io, const char *_name, bool
     init(hdr, isTLK);
 }
 
-CJHTreeNode *CDiskKeyIndex::loadNode(CJHTreeNode * optNode, offset_t pos)
+CJHTreeNode *CDiskKeyIndex::loadNode(cycle_t * fetchCycles, CJHTreeNode * optNode, offset_t pos)
 {
     nodesLoaded++;
     unsigned nodeSize = keyHdr->getNodeSize();
     MemoryAttr ma;
     char *nodeData = (char *) ma.allocate(nodeSize);
-    MTIME_SECTION(queryActiveTimer(), "JHTREE read node");
+
+    CCycleTimer fetchTimer(fetchCycles != nullptr);
```

Comment:
I'm not sure if I should keep this call to MTIME_SECTION.  I think the stats coming back are likely to be much more useful.  Thoughts?

Reply:
I say keep the MTIME_SECTION(queryActiveTimer(), "JHTREE read node");

Reply:
After more discussion I think removing some MTIME_SECTION() calls would be good, since they have a CritSec and a find().